### PR TITLE
Feat/autogen

### PR DESCRIPTION
**Pull request by jleonatti**
- Add `_build.py` to generate operator Python classes from `_operator_list.py`.
- The `_operator_list.py` file is generated from the operators within `dpf.html`.
- Clean up whitespace in non-autogenerated files within the operators directory.
- A handful of regenerated operator classes are added to this draft PR for review.

Still to be done:
- [ ]  Don't think operator classes are functional as is. Refactor needed there.
- [x] Since operator classes are now pascal case instead of snake case to follow the Python convention, a deprecation warning needs to be added.
- [ ] Incorporate new API to get list of operators instead of using html file.
- [ ] Add readme on how to generate these files.

### CODE REVIEW

**Comment by akaszynski** [comment]
If not all the operators are available by scraping the html, we need a reliable way of inspecting or querying the server for the available operators.  @rlagha, do we have a public operator that does this?

File: ansys/dpf/core/operators/averaging/elemental_difference_fc.py [hunk 1]
```diff
@@ -90,36 +106,35 @@ def _spec():
...
-        layers if true (default is false)""",
+                    document="""The max elemental difference is taken through
+        the different shell layers if true
+        (default is false)""",
                 ),
             },
             map_output_pin_spec={
                 0: PinSpecification(
                     name="fields_container",
-                    type_names=['fields_container'],
```
Comment (cbellot000) [inline ansys/dpf/core/operators/averaging/elemental_difference_fc.py]: @jleonatti this is needed to work I think

File: ansys/dpf/core/operators/averaging/elemental_difference_fc.py [hunk 2]
```diff
@@ -283,14 +311,13 @@ def fields_container(self):
...
 
         Examples
         --------
         >>> from ansys.dpf import core as dpf
         >>> op = dpf.operators.averaging.elemental_difference_fc()
         >>> # Connect inputs : op.inputs. ...
-        >>> result_fields_container = op.outputs.fields_container() 
-        """
+        >>> result_fields_container = op.outputs.fields_container()
+        """  # noqa: E501
```
Comment (cbellot000) [inline ansys/dpf/core/operators/averaging/elemental_difference_fc.py:322]: ?

**Comment by jleonatti** [comment]
> If not all the operators are available by scraping the html, we need a reliable way of inspecting or querying the server for the available operators. @rlagha, do we have a public operator that does this?

@cbellot000 has exposed an API to get a list of operators. I have not implemented that here yet.

File: ansys/dpf/core/operators/averaging/elemental_difference_fc.py (hunk 1 above)
Comment (jleonatti) [reply ansys/dpf/core/operators/averaging/elemental_difference_fc.py]: I believe this is fixed now!

**Comment by jleonatti** [comment]
This work is added to the dpf-python-core repo and will be migrated here later.

**Pull request closed without merging**